internal/controller/object/object.go

```diff
@@ -113,7 +114,15 @@ func Setup(mgr ctrl.Manager, o controller.Options, sanitizeSecrets bool, pollJit
 		}),
 		managed.WithFinalizer(&objFinalizer{client: mgr.GetClient()}),
 		managed.WithPollInterval(o.PollInterval),
-		managed.WithPollJitterHook(pollJitter),
+		managed.WithPollIntervalHook(func(mg resource.Managed, pollInterval time.Duration) time.Duration {
+			if mg.GetCondition(xpv1.TypeReady).Status == v1.ConditionTrue {
+				// This is the same as runtime default poll interval with jitter, see:
+				// https://github.com/crossplane/crossplane-runtime/blob/7fcb8c5cad6fc4abb6649813b92ab92e1832d368/pkg/reconciler/managed/reconciler.go#L573
+				return pollInterval + time.Duration((rand.Float64()-0.5)*2*float64(pollJitter)) //#nosec G404 -- no need for secure randomness
```

Comment:
```suggestion
				return pollInterval + time.Duration((rand.Float64()-0.5)*2*float64(pollJitter)) //nolint G404 // No need for secure randomness
```

Nit: let's use golint style disable comments here (I know this was just copied from upstream).

Reply:
Done. Also, applied jitter to 30 seconds as well.